src/main/webapp/script.js

```diff
@@ -173,46 +160,147 @@ function getSchoolInfo() {
       let chart = new google.visualization.PieChart
           (document.getElementById('race-piechart'));
       chart.draw(data, options);
-    }
+}
 
-    function drawGenderChart() {
-      let data = google.visualization.arrayToDataTable([
-        ['Gender', 'Percentage'],
-        ['Men', numMen],
-        ['Women', numWomen],
-      ]);
+/**
+ * Creates Donut Pie Chart displaying gender breakdown.
+ */
+function drawGenderChart(numMen, numWomen) {
+  let data = google.visualization.arrayToDataTable([
+      ['Gender', 'Percentage'],
+      ['Men', numMen],
+      ['Women', numWomen],
+  ]);
 
-      let options = {
-        title: 'Breakdown by Gender',
-        pieHole: 0.4,
-        colors: ['#D4EBD0', '#A4C5C6'],
-      };
+  let options = {
+      title: 'Breakdown by Gender',
+      pieHole: 0.4,
+      colors: ['#D4EBD0', '#A4C5C6'],
+  };
 
-      let chart = new google.visualization.PieChart
-          (document.getElementById('gender-piechart'));
-      chart.draw(data, options);
+  let chart = new google.visualization.PieChart
+      (document.getElementById('gender-piechart'));
+  chart.draw(data, options);
+}
+
+ /**
+ * Finds and returns the main campus from a list of main and satellite campuses
+ * of the same name.
+ * @param {!Array<object>} schools This is the array of school objects returned
```

Comment:
"Object" type uses an Uppercase "O" and requires an nullability indicator (! or ?). Fix throughout.